src/server/db_slice.cc

```diff
@@ -495,7 +495,8 @@ OpResult<pair<PrimeConstIterator, unsigned>> DbSlice::FindFirstReadOnly(const Co
   return OpStatus::KEY_NOTFOUND;
 }
 
-DbSlice::AddOrFindResult DbSlice::AddOrFind(const Context& cntx, string_view key) noexcept(false) {
+OpResult<DbSlice::AddOrFindResult> DbSlice::AddOrFind(const Context& cntx,
+                                                      string_view key) noexcept(false) {
```

Comment:
can we change this to `noexcept(true)` now? or just `noexcept`?
Or just remove this, as we never throw in other places anyway?

Reply:
just `noexcept` will suffice but if we omit it completely it's also ok I guess